source/developers-guide/cheat-sheet/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Cheat-Sheet for developers
+github_link: developers-guide/cheat-sheet/index.md
+shopware_version: 5.3.0
+tags:
+  - cheat-sheet
+  - developers
+  - beginner
+indexed: true
+group: Developer Guides
+subgroup: General Resources
+menu_title: Cheat-Sheet
+menu_order: 1
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+
+This article will give you brief information about often used functionalities, methods and good practises for developers, who are working on shopware.
+<div class="alert alert-warning">
+This list is not complete. If you miss something important, feel free to open a pull request with the GitHub link in the top.
+</div>
+
+## Templating
+Have a look here for a <a href="http://community.shopware.com/files/downloads/templatecheatsheeten-12249471.pdf" target="_blank">Smarty Cheat-Sheet</a>
+
+## Events and hooks
+Use your main plugin class or a subscriber class which implements the `\Enlight\Event\SubscriberInterface` to register new events or hooks
+
+### Events
+```php
+public static function getSubscribedEvents()
+{
+    return [
+        'EVENT_NAME1' => 'EVENT_LISTENER1',
+        'EVENT_NAME2' => ['EVENT_LISTENER2', POSITION],
+        'EVENT_NAME3' => [
+            ['EVENT_LISTENER3_0'],
+            ['EVENT_LISTENER3_1', POSITION],
+        ],
+    ];
+}
+```
+
+### Hooks
+types: before / after / replace
+```php
+public static function getSubscribedEvents()
+{
+    return [
+        'CLASS::FUNCTION::TYPE' => 'LISTENER',
+    ];
+}
+```
+
+## Attributes
+### Creating a new attribute
```

Comment:
I think you should add a hint that this can also be used to update existing attributes :)